documentation/src/main/markdown/packagemgrs/opam.md

```diff
@@ -0,0 +1,37 @@
+# Opam Support
+
+## Related properties
+
+[Detector properties](../properties/detectors/opam.md)
+
+[detect_product_short] has two detectors for Opam:
+
+* OPAM Build Detector
```

Comment:
I looked at this page: https://documentation.blackduck.com/bundle/detect/page/components/detectors.html

The trend in our Detector names would be to use "Opam CLI" and Opam Lock".